changelog.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+- Declare this extension safe for parallel reading
+
+## [0.1.5] - 2018-05-15
+
+[Unreleased]: https://github.com/sphinx-contrib/autoprogram/compare/v1.0.0...HEAD
+[0.1.5]: https://github.com/sphinx-contrib/autoprogram/compare/0.1.4...0.1.5
```

Comment:
Could you move them to *docs/changes.rst*?

Reply:
Will do.